src/action/invoice.js

```diff
@@ -61,8 +61,10 @@ class InvoiceAction {
   async generateUri() {
     try {
       const { invoice, settings } = this._store;
+      const amount = toSatoshis(invoice.amount, settings);
+      this.checkAmount({ satAmount: amount });
```

Comment:
We should do an early return in case `checkAmount()` returns false and there isn't enough inbound liquidity. Otherwise what's the value of displaying an invoice to the user?

Reply:
It might be fun for people who just want to "try out" the app :thinking:

Reply:
That's true. I'm fine with just displaying a warning for now.